src/main/steps/common/page.njk

```diff
@@ -32,9 +32,11 @@
     },
     html: feedback + languageToggle
   }) }}
+  {% block backLink %}
   {% if not hideBackButton %}
     <a href="#" class="govuk-back-link">{{ back }}</a>
   {% endif %}
+  {% endblock %}
```

Comment:
Note: I've removed this `if not hideBackButton` in a separate PR [#279](https://github.com/hmcts/nfdiv-frontend/pull/279/files#diff-96e7ff997b419a4da421faea6531b665d4991a62052ee8928301f3ce1a23b25b)